inline.go

```diff
@@ -728,25 +715,38 @@ func linkEndsWithEntity(data []byte, linkEnd int) bool {
 	return entityRanges != nil && entityRanges[len(entityRanges)-1][1] == linkEnd
 }
 
+var prefixes = [][]byte{
```

Comment:
This variable name was okay when it was inside `maybeAutoLink` func, but it's not okay as a package level variable. It's not descriptive at all.

Call it `autoLinkPrefixes` maybe.